pkg/operator2/secret.go

```diff
@@ -49,11 +82,12 @@ func newSessionSecretsJSON() []byte {
 		},
 		Secrets: []legacyconfigv1.SessionSecret{
 			{
-				Authentication: string(randomBits(sha256KeyLenBits)), // TODO these lengths are probably wrong
-				Encryption:     string(randomBits(aes256KeyLenBits)),
+				Authentication: string(randomBits(sha256KeyLenBits)), // 64 chars
```

Comment:
Still need to use `base64.RawURLEncoding.EncodeToString` instead of blind cast to `string`.

Reply:
got it